.github/workflows/tests.yml

```diff
@@ -29,6 +29,11 @@ jobs:
           override: true
           components: rustfmt, clippy
 
+      - name: install php
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 8.1
```

Comment:
You should setup composer cache too

Reply:
composer deps should be cached, as they are in target/third-party/*/vendor.

Reply:
Yeah but it's not current because you cache the final version and not composer's cache
If you have a cache for composer modules used between different projects would be downloaded once